src/lava/lib/dl/netx/hdf5.py

```diff
@@ -91,59 +95,60 @@ def get_neuron_params(
             The Lava process that implements the neuron described.
         """
         neuron_type = neuron_config['type']
+        num_message_bits = None
+        if 'messageBits' in neuron_config.keys():
+            num_message_bits = neuron_config['messageBits']
         if neuron_type in ['LOIHI', 'CUBA']:
+            if num_message_bits is None:
+                num_message_bits = 0  # default value
             neuron_process = LIF
-            neuron_params = {
-                'neuron_proc': neuron_process,
-                'vth': neuron_config['vThMant'],
-                'du': neuron_config['iDecay'] - 1,
-                'dv': neuron_config['vDecay'],
-                'bias_exp': 6,
-                'use_graded_spikes': False,
-            }
+            neuron_params = {'neuron_proc': neuron_process,
+                             # Key that actually represents bias in the
+                             # neuron_process
```

Comment:
comment is confusing, is there other argument that does not represent the bias but suggests it does?

Reply:
It's because some the LIF process use `bias_mant` whereas sigma delta processes use `bias`. Typically in ucode neurons, it's efficient to represent the bias value itself. That saves a bit shifting operation. But I get your point. Reworded the comment to 
`# Map bias parameter in hdf5 config to bias_mant`